internal/oci/runtime_vm.go

```diff
@@ -540,7 +540,7 @@ func (r *runtimeVM) DeleteContainer(c *Container) error {
 		return err
 	}
 
-	if _, err := r.task.Shutdown(r.ctx, &task.ShutdownRequest{ID: c.ID()}); err != nil {
+	if _, err := r.task.Shutdown(r.ctx, &task.ShutdownRequest{ID: c.ID()}); err != nil && !errors.Is(err, ttrpc.ErrClosed) {
```

Comment:
might be nice to have a code comment here on why ttrpc.ErrClosed is special.

Reply:
I can add a comment, but that's well detailed in the commit message.
Let me know your preference, but I have to admit I'm afraid of touching this PR again as, for the first time, CI seems to have passed. :-)